File: C_Programming/Practise_Programs/PP_070.c
/*
==============================================================================
 File Name   : PP_070.c

 Program     : Display Array Elements Using Array Indexing

 Description : Demonstrates how to access and display array elements using
               array indexing. The array is passed to a function and each
               element is accessed using its index.

 Author      : Shubham Gadhe

 Time Complexity  : O(1)
 Space Complexity : O(1)
==============================================================================
*/

#include<stdio.h>

/*
==============================================================================
 Function Name : Display

 Description   : Displays the elements of the array using array indexing.

 Input         : Integer array
 Output        : None
==============================================================================
*/

void Display(int Arr[])
{
    // Display all array elements using array indexing
    printf("%d\n", Arr[0]);      
    printf("%d\n", Arr[1]);
    printf("%d\n", Arr[2]);
    printf("%d\n", Arr[3]);
    printf("%d\n", Arr[4]);
}

/*
==============================================================================
 Entry Point Function
==============================================================================
*/

int main()
{
   int Brr[5] = {10,20,30,40,50};  

   // Pass the array to the function
   Display(Brr);    

   return 0;  
}

/*
==============================================================================
 Sample Output
------------------------------------------------------------------------------
10
20
30
40
50
==============================================================================
*/

File: Java_Programming/Practise_Programs/PP_070.java
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_070
//
//  Description       : This program accepts the number of rows
//                      and columns from the user and displays
//                      uppercase alphabets starting from 'A'
//                      in each row.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 08/08/2026
//
//  Time Complexity   : O(r × c)
//  Space Complexity  : O(1)
//
//////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Pattern
{
    //////////////////////////////////////////////////////////////////
    //
    //  Function Name    : Display()
    //
    //  Description      : It is used to display uppercase
    //                     alphabets starting from 'A'
    //                     up to the specified number of
    //                     columns in each row.
    //
    //  Author           : Shubham Somanath Gadhe
    //  Date             : 08/08/2026
    //
    //////////////////////////////////////////////////////////////////

    public void Display(int iRow, int iCol)
    {
        int i = 0, j = 0;
        char ch = '\0';

        for(i = 1; i <= iRow; i++)
        {
            for(j = 1, ch = 'A'; j <= iCol; j++, ch++)
            {
                System.out.print(ch + "\t");
            }

            System.out.println();
        }
    }
}

class PP_070
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0, iValue2 = 0;

        System.out.println("Enter the number of Rows: ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter the number of Column: ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue1, iValue2);
    }
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input  :
//      Enter the number of Rows :
//      4
//
//      Enter the number of Column :
//      4
//
//  Sample Output :
//
//      A    B    C    D
//      A    B    C    D
//      A    B    C    D
//      A    B    C    D
//
//////////////////////////////////////////////////////////////////